emails/templates/emails/wrapped_email.html

```diff
@@ -39,6 +39,9 @@
               <td width="100%" align="center" style="line-height: 150%;">
                 <p style="color: #363959; font-size: 13px; font-family: sans-serif; margin-top: 0; margin-bottom: 0;">
                   This message was forwarded from <span style="font-family: sans-serif; font-weight: bolder; color: #20123a; text-decoration: none; font-size: 13px;">{{ display_email|safe }}</span> by <a href="{{ SITE_ORIGIN }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">Firefox Relay</a>.
+                  {% if has_attachment %}
+                    <br>Emails with a maximum size of <b>150KB</b> have attachments support. To learn more check our <a href="{{ faq_page }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">FAQ page</a>.
```

Comment:
Suggest changing to match what's in the dashboard footer (and removing extra `<br>` tag ...
```suggestion
                    Firefox Relay supports email forwarding (including attachments) of email up to 150KB in size. To learn more check our <a href="{{ faq_page }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">FAQ page</a>.
```

Reply:
The `<br>` was to ensure that this statement was in a new line as we did [previously](https://github.com/mozilla/fx-private-relay/pull/613/files#diff-8d6893341f0957c36ae14bd59cd9825dL42-L44). Is there a better way to handle this?

Reply:
Ah, good call. Nvm me.